CHANGELOG.md

```diff
@@ -1,6 +1,81 @@
 # Lotus changelog
 
-# UNRELEASED
+# v 1.25.0-rc1 / 2023-10-17
+
+The first release candidate of 1.25 optional upgrade includes the deliverables to 1.23.4 and 1.24.0-mandatory. 
```

Comment:
I think we should drop all references to 1.23.4 here. Instead, just treat this as the 1.25.0-rc1, that's being provided on top of 1.24.0-rc2.

Reply:
(since there will never be a 1.23.4)